src/model.coffee

```diff
@@ -51,6 +56,13 @@ class window.Brainstem.Model extends Backbone.Model
 
 
   #
+  # Init
+
+  constructor: (options = {}) ->
+    super
+    @storageManager = StorageManager.get()
```

Comment:
As mentioned in our previous discussions, we should consider making this a global so there isn't a circular dependency b/w our core data objects and the storage manager.

Probably a task we can do as part of a later PR effort.